dbms/programs/server/MySQLHandler.cpp

```diff
@@ -221,7 +221,8 @@ void MySQLHandler::authenticate(const String & user_name, const String & auth_pl
     {
         // For compatibility with JavaScript MySQL client, Native41 authentication plugin is used when possible (if password is specified using double SHA1). Otherwise SHA256 plugin is used.
         auto user = connection_context.getUser(user_name);
-        if (user->authentication.getType() != DB::Authentication::DOUBLE_SHA1_PASSWORD)
+        const DB::Authentication::Type user_auth_type = user->authentication.getType();
```

Comment:
Why there is something connected with authentication in this pr?
Can you please remove it, and make a separate pr?

Reply:
This looks like a merge gone wrong, removed that already